Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.yml

```diff
@@ -165,12 +171,6 @@ script:
       required: false
       secret: false
     deprecated: false
-    description: Push modified IOCs to Cortex XDR.
-    execution: false
-    name: xdr-iocs-push
-  - arguments:
-    - default: false
-      description: The time of the file creation (use UTC time zone).
```

Comment:
Why was this removed?

Reply:
not purposely (and the yaml is broken), fixing